app/lib/server/functions/validateName.js

```diff
@@ -0,0 +1,9 @@
+import { Meteor } from 'meteor/meteor';
+
+import { settings } from '../../../settings';
```

Comment:
if you cannot import from the final file, pls try at least import from the `server` folder, for example:
```suggestion
import { settings } from '../../../settings/server';
```